corehq/apps/locations/templates/locations/manage/location.html

```diff
@@ -20,13 +20,10 @@
 {% block js-inline %}{{ block.super }}
     <script type="text/javascript">
         var LOAD_LOCS_URL = '{{ api_root }}';
-
-        // for product and user per location selection
-        $(function () {
-            $("#id_users-selected_ids").width(800).height(400).multiselect();
-            $("#id_products-selected_ids").width(800).height(400).multiselect();
-        });
+            NEW_MOBILE_WORKER_URL = '{% url "new_mobile_worker_modal" domain %}';
+            LOCATION_ID = {{ location.get_id|JSON }};
```

Comment:
I like moving js into its own dedicated files, but it seems a little fragile that location.js is dependent on these variables. Any thoughts? (I don't have any beyond maybe commenting in location.js that it's dependent on this file.)